src/status_im/native_module/core.cljs

```diff
@@ -278,6 +279,52 @@
              :connection-string connection-string})
   (.inputConnectionStringForBootstrapping ^js (status) connection-string config-json callback))
 
+(defn deserialize-and-compress-key
+  "Provides a community id public key to status-go and gets back a key that status-mobile understands"
+  [desktop-key callback]
+  (log/info "[native-module] Compressing public key"
+            {:fn          :convert-desktop-key-to-mobile-key
+             :desktop-key desktop-key})
+  (.deserializeAndCompressKey ^js (status) desktop-key callback))
+
+
+(defn serialise-public-key-into-compressed-key
+  "Provides public key to status-go and gets back a compressed key"
+  [public-key callback]
+  (let [serialisation-key constants/serialization-key
+        multi-code-prefix constants/multi-code-prefix
+        multi-code-key    (str multi-code-prefix (subs public-key 2))]
+    (log/info "[native-module] Compressing public key"
+              {:fn             :serialise-public-key-into-compressed-key
+               :public-key     public-key
+               :multi-code-key multi-code-key})
+    (.multiformatSerializePublicKey ^js (status) multi-code-key serialisation-key callback)))
+
+(defn deserialise-compressed-key-into-public-key
```

Comment:
In Clojure there's a very strong convention of naming functions that do type/format conversions using *arrows*. See in the Clojure Style Guide (https://github.com/bbatsov/clojure-style-guide#conversion-functions).

Suggestions:

- Rename `deserialise-compressed-key-into-public-key` to `compressed-key->public-key`
- Rename `serialise-public-key-into-compressed-key` to `public-key->compressed-key`.
- Remove the docstrings from `deserialise-compressed-key-into-public-key` and `serialise-public-key-into-compressed-key` because they seem redundant, especially after the suggested renames. 

Interestingly after renaming we get to see one function is the reverse of the other. Notice I dropped the `serialize`/`deserizalize` words, as this is less important IMO after the renames.

Reply:
The method `deserialise-compressed-key-into-public-key` explains that we are achieving compressed-key->public-key by deserialization. renaming it to `compressed-key->public-key` will mean we are getting a->b , but we will lose the how.. should we then keep the docstrings and rename as you suggested ? what do you think ?

Reply:
Great question! In general, I think functions should never explain the *how*, only the *what*. The *how* is always on the implementation side or in the docstrings/comments. 

> should we then keep the docstrings and rename as you suggested

Precisely @siddarthkay, if you feel like it's important to give more information about *how* something works that's not clear by the code itself, then a comment/docstring works wonders.